test/regress/regress1/sygus/repair-const-unk.sy

```diff
@@ -0,0 +1,17 @@
+; EXPECT: unknown
+; COMMAND-LINE: --sygus-out=status --sygus-repair-const --lang=sygus2 --no-sygus-pbe
```

Comment:
Why do you need `--no-sygus-pbe` here?

Reply:
That just happened to be the options I was using. This isn't important really.

Reply:
I think it's best to have the minimal set of options that leads to desired behavior so that we avoid confusion in the future. If you can remove `--no-sygus-pbe` I think it's best to do it.